doc/rbd/rbd-exclusive-locks.rst

```diff
@@ -0,0 +1,78 @@
+.. _rbd-exclusive-locks:
+
+====================
+ RBD Exclusive Locks
+====================
+
+.. index:: Ceph Block Device; RBD exclusive locks; exclusive-lock
+
+Exclusive locks are a mechanism designed to prevent multiple processes
+from accessing the same Rados Block Device (RBD) in an uncoordinated
+fashion. Exclusive locks are heavily used in virtualization (where
+they enable live migration, among other things), and also in RBD
```

Comment:
Nit: technically live-migration works just fine w/o the `exclusive-lock` feature being enabled.

Reply:
Gotcha. Will drop that reference.